language/move-vm/test-utils/src/gas_schedule.rs

```diff
@@ -60,6 +60,9 @@ pub const STRUCT_SIZE: AbstractMemorySize = AbstractMemorySize::new(2);
 /// For exists checks on data that doesn't exists this is the multiplier that is used.
 pub const MIN_EXISTS_DATA_SIZE: AbstractMemorySize = AbstractMemorySize::new(100);
 
+/// Generic gas cost table scale factor, that will be applied linearly.
+pub const TABLE_GAS_COST_SCALE_FACTOR: u64 = 1;
```

Comment:
Maybe `float` would be more practical, but then we would have to convert values before multiplication and then convert those back to `u64`...

Let's try with `u64` for now, we can change it later if needed 👍

Reply:
Done :+1: